server/default_config.py

```diff
@@ -76,15 +76,6 @@
     # A list of allowed matrix types.  If an empty list, then all matrix types are allowed
     allowed_matrix_types: []
 
-    matrix_cache:
-      # The maximum number of datasets that may be opened at one time.  The least recently used dataset
-      # is evicted from the cache first.
-      max_datasets: 5
-
-      # A matrix is automatically removed from the cache after timelimit_s number of seconds.
-      # If timelimit_s is set to None, then there is no time limit.
-      timelimit_s: 30
-
   single_dataset:
```

Comment:
we can convene later on removing everything related to single dataset mode

Reply:
good cleanup task

Reply:
I gotcha covered: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-explorer/181